clients/drcachesim/tests/decode_cache_test.cpp

```diff
@@ -35,6 +35,10 @@
 #include <iostream>
 #include <vector>
 
+// Needs to be included before memref.h or build_target_arch_type() will not
```

Comment:
@derekbruening Do we have any preexisting issue for this (I couldn't find one). We probably should clean this up.

Reply:
I don't remember hitting this issue: did we hit it before?  Maybe nobody hit any problem before and so nobody knew to file an issue.

Reply:
Filed #7236